README.md

```diff
@@ -140,7 +141,8 @@ Right side from SATA port is completed, all components soldered:
 ![](https://github.com/badger707/m920q-dual-NVME/blob/main/pictures/IMG_5894_.PNG)
 
 
-Left side from SATA port is completed, all components are soldered. Yes I missed one cap and thats intentional -- I didnt wanted to take any risk and melt the SATA port, cap is just too close to it. So I skipped C377 and placed C379 to its place:<br>
+Left side from SATA port is completed, all components are soldered. Yes I missed one cap and thats intentional -- I didnt wanted to take any risk and melt the SATA port, cap is just too close to it. So I skipped C379 and placed C377 to its place:<br>
+<b>Note numbering on picture is incorrect - capacitors are labelled going downwards as follows: C378, C377, 379</b>
```

Comment:
not quite right, it does not matter in which order they go

Reply:
Ah, I see - they're all connected in parallel.